addons/hr_holidays/static/src/im_status_patch.xml

```diff
@@ -1,10 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
     <t t-inherit="mail.ImStatus" t-inherit-mode="extension">
-        <xpath expr="//span" position="inside">
+        <xpath expr="//*[@name='icon']" position="replace">
             <i t-if="props.persona.im_status === 'leave_online'" class="fa fa-plane text-success" title="Online" role="img" aria-label="User is online"/>
             <i t-if="props.persona.im_status === 'leave_away'" class="fa fa-plane text-warning" title="Idle" role="img" aria-label="User is idle"/>
             <i t-if="props.persona.im_status === 'leave_offline'" class="fa fa-plane text-700" title="Out of office" role="img" aria-label="User is out of office"/>
-        </xpath>        
+            <t t-else="">$0</t>
```

Comment:
The whole chain of conditions above need to be `elif` instead of `if` to make the `else` work as intended.
But it's better to avoid `replace` and use `inside` again instead, adding extra condition is less error prone (like this one). Just the base case needs to be adapted to work conditionally, not triggering naturally in the `else` condition.

Reply:
I think it's good to keep `elif` chain, so that all of this acts like a `switch/case` and we are confident that we won't have bug to show 2 icons at once.

Also the fallback would leak some technical details, such as `im_partner` as a value for this `im_status`. And if we introduce a new `im_status`, it's much easier to see "no IM status" and grep it to change the feature than to show nothing at all.